src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -53,7 +54,7 @@ export class MdSortableTable extends React.PureComponent {
     }
 
     render() {
-        const { defaultColWidths } = this.props;
+        const { subTitle, defaultColWidths, maxContainerHeight } = this.props;
```

Comment:
Codacy found an issue: ['subTitle' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)